tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/NUnit/IWorkItem.cs

```diff
@@ -2,61 +2,61 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
 
 using System;
 using System.Reflection;
 using Datadog.Trace.DuckTyping;
 
-namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.NUnit
```

Comment:
Should the other `ITest` etc props below be nullable? 🤔

Reply:
done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)